tests/unit/Manifest/ManifestLoaderTest.php

```
@@ -116,6 +117,39 @@ public function testExternalManifestLoadsFileMappings(): void
+        $package->method('getName')->willReturn('yii2-extensions/bad');
+
+        try {
+            $this->expectException(RuntimeException::class);
+            $this->expectExceptionMessage('could not read manifest file');
+
+            (new ManifestLoader(new ManifestSchema()))->load($package, $providerPath);
+        } finally {
+            @unlink($manifestPath);
+            @rmdir($providerPath);
+        }
+    }
```

PHPMD (2.15.0)

[error] Remove error control operator '@' on line 148. (undefined)

(ErrorControlOperator)